app/views/logs_searches/_error_message.html.erb

```diff
@@ -0,0 +1,3 @@
+<% if field_error(@search, key).present? %>
+  <p class="govuk-body govuk-error-message">Please <%= message %></p>
```

Comment:
[GOV Design System](https://design-system.service.gov.uk/components/error-message/#be-clear-and-concise) says "Do not use ‘please’ because it implies a choice".

```suggestion
  <p class="govuk-body govuk-error-message"><%= message %></p>
```

(not sure if you'll need to mess with capitalisation because of this)

Reply:
👍